### PR TITLE
Web player guide

### PR DESCRIPTION
**Pull request by Enriqe**
Adds web player homepage to centralize guides and links to other pages. 

The go.amp.dev/web-player link (introduced in https://github.com/ampproject/amp.dev/pull/5767) will point to this page when it's live.

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Can you rebase against `future`?

File: pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+You can think similarly of the web player. By only linking to a single story, you can miss out on a number of features and experiences that are found in other story platforms (e.g. Instagram, Snapchat) and are natural to story users. See table below for a few examples.
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td><strong>Single Story</strong>
+   </td>
+   <td><strong>Web player</strong>
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md:29]:
```suggestion
   <td><strong>Web Story Player</strong>
```

File: pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md [hunk 2]
```diff
@@ -0,0 +1,387 @@
+---
+$title: Web Story Player - User’s guide
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md:2]:
```suggestion
$title:  Introduction to Web Story Player on non-AMP pages
```

File: pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+formats:
+  - websites
+  - stories
+---
+
+# Web Story Player - User’s guide
+
+The Web Player has significantly matured (thanks for all the feedback and insights), and is in a great place for publishers, developers, tools and platforms to start using in their site experiences. This update recaps key capabilities from the Web Story Player, and provides guides on common implementation patterns.
+
+
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md:13]:
```suggestion


```

File: pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+## What is the Web Player?
+
+{{ image('/static/img/docs/web-player-main.gif', 750, 625,   align='center', caption='Web Story Player demo' alt='web player demo' ) }}
+
+The Web Story Player is the best way for you to give users a rich user experience on your sites using Web Stories.  Think of it like a video player: having a single video in your device is fine, but put it in a video player live on the web, and it unlocks a number of features and benefits: There’s additional controls (e.g. captions), it lives among millions of other videos that can provide related content at the end of your original video – which adds discoverability, and more.
+
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md:19]: Inset note here about non-AMP vs AMP and direct readers to the embedding WST in AMP pages doc

File: pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+1. Declaring them as anchor tags inside the player HTML.
+2. Using the fetch behavior to fetch more stories from an endpoint.
+3. Using the add() API to add a story or multiple stories.
+
+If you only want to display a static set of stories in your player, your best option is (1). But if you want to dynamically add stories, you can do so with options (2) or (3).
+
+
+### Display stories
+
+The simplest way to display and add stories to the player is to declare them directly in the HTML where the `<amp-story-player>` element is located as &lt;a> tags. See code sample in the “[Display a Web Story](https://amp.dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp/?format=stories#display-a-web-story)” section of the guide.
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md:173]:
```suggestion
The simplest way to display and add stories to the player is to declare them directly in the HTML where the `<amp-story-player>` element is located as `<a>` tags. See code sample in the “[Display a Web Story](https://amp.dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp/?format=stories#display-a-web-story)” section of the guide.
```

File: pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md [hunk 6]
```diff
@@ -0,0 +1,387 @@
...
+<table>
+  <tr>
+    <td><img src="https://github.com/ampproject/amphtml/blob/main/src/amp-story-player/img/close-button-left.png?raw=true"></td>
+    <td><img src="https://github.com/ampproject/amphtml/blob/main/src/amp-story-player/img/close-and-skip-next.png?raw=true"></td>
+<td><img src="https://github.com/ampproject/amphtml/blob/main/src/amp-story-player/img/close-button-custom-background.png?raw=true"></td>
+  </tr>
+</table>
+
+
+To configure them, specify a JSON configuration with the `type=”application/json”` attribute as a child of the `&lt;amp-story-player>` element.
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md:230]:
```suggestion
To configure them, specify a JSON configuration with the `type=”application/json”` attribute as a child of the `<amp-story-player>` element.
```

File: pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md (hunk 4 above)
Comment (Enriqe) [reply pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md:15]: Added it below under a `[tip type="note"]`

File: pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md [hunk 7]
```diff
@@ -0,0 +1,387 @@
...
+   </td>
+  </tr>
+</table>
+
+## Contents
+
+
+[TOC]
+
+
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md:119]:
```suggestion
```

**Review by CrystalOnScript — APPROVED** [review]
Just one nit, otherwise LGTM!

**Comment by Enriqe** [comment]
@sebastianbenz could you help us deploy this page so that it's live on amp.dev? We might start getting some visits soon via the short link.

**Comment by sebastianbenz** [comment]
@Enriqe I've triggered a deploey. Should be live in 45-60 min.

**Comment by Enriqe** [comment]
Thank you!

**Pull request merged**